template/checknothing/template.proto

```diff
@@ -23,5 +23,15 @@ option (istio.mixer.v1.config.template.template_variety) = TEMPLATE_VARIETY_CHEC
 // CheckNothing represents an empty block of data that is used for Check-capable
 // adapters which don't require any parameters. This is primarily intended for testing
 // scenarios.
+//
+// Example config:
+// ```
+// apiVersion: "config.istio.io/v1alpha2"
+// kind: checknothing
+// metadata:
+//   name: denyrequest
+//   namespace: istio-config-default
```

Comment:
Is it worth changing to a random namespace since there're still discussions happening around the use of a special namespace?

Reply:
Are we planning to change the namespace for 0.2 ? This is to help operators write configs for 0.2, I would like to get this PR in and also the associated docs for 0.2.

Reply:
we no longer want to use `istio-config-default` namespace in examples.
However, that should go in as a single `s/istio-config-default/istio-config-default` replace.